jenkins/artifacts/jenkinsfile

```diff
@@ -22,9 +23,7 @@ pipeline {
         RELEASE =sh (returnStdout: true, script: """
         echo \"${params.RELEASE}\"
         """).trim()
-        BRANCH =sh (returnStdout: true, script: """
-        echo \"${params.BRANCH}\"
-        """).trim()
+        BRANCH = getBranchName(env.GIT_BRANCH, params.BRANCH)
```

Comment:
Why do we need to check between env and params? Why don't we keep it just read from params?

Reply:
since we use same jenkins file, we have to add a logic to get dev branch when we run this jenkins file in a multibranch pipeline jobs.  GIT_BRANCH is an inbuilt env variable which will return dev branch info. hence i have added this